gst/nnstreamer/nnstreamer_plugin_api_impl.c

```diff
@@ -747,9 +762,13 @@ static GstCaps *
 _get_tensors_caps (const GstTensorsConfig * config)
 {
   GstCaps *caps;
+  GstStructure *structure;
 
   caps = gst_caps_from_string (GST_TENSORS_CAP_DEFAULT);
 
+  /* structure for backward compatibility */
```

Comment:
How about adding a null check (config->info)?
dim_str(784 and 775) can be null if config->info is null.

Reply:
updated it
Thank you :)